ts/WoltLabSuite/Core/Acp/Ui/User/Action/DisableAction.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * @author  Joshua Ruesweg
+ * @copyright  2001-2021 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Acp/Ui/User/Action
+ * @since       5.5
+ */
+
+import * as Ajax from "../../../../Ajax";
+import * as Core from "../../../../Core";
+import { AjaxCallbackObject, AjaxCallbackSetup, DatabaseObjectActionResponse } from "../../../../Ajax/Data";
+import * as UiNotification from "../../../../Ui/Notification";
+import AbstractUserAction from "./AbstractUserAction";
+import * as EventHandler from "../../../../Event/Handler";
+
+export class DisableAction extends AbstractUserAction implements AjaxCallbackObject {
+  protected init(): void {
+    this.button.addEventListener("click", (event) => {
+      event.preventDefault();
+      const isEnabled = Core.stringToBool(this.userDataElement.dataset.enabled!);
+
+      Ajax.api(this, {
+        actionName: isEnabled ? "disable" : "enable",
+      });
+    });
+  }
+
+  _ajaxSetup(): ReturnType<AjaxCallbackSetup> {
+    return {
+      data: {
+        className: "wcf\\data\\user\\UserAction",
+        objectIDs: [this.userId],
+      },
+    };
+  }
+
+  _ajaxSuccess(data: DatabaseObjectActionResponse): void {
+    if (data.objectIDs.includes(this.userId)) {
```

Comment:
Have you made sure that this works without `mysqlnd` (string vs integer)? (Also applies to other places with same code!)